app/helpers/categories_helper.rb

```diff
@@ -0,0 +1,5 @@
+module CategoriesHelper
+  def category
+    @categories = Category.all
```

Comment:
không lấy toàn bộ record thế này. phân trang để đảm bảo hiệu năng (will_paginate, kaminari, pagy tuỳ chọn 1 trong 3)